config/locales/forms/find_support.en.yml

```diff
@@ -0,0 +1,59 @@
+en:
+  find_support:
+    defaults:
+      options:
+        yes: "Yes"
+        soon: "I might be soon"
+        no: "No"
+        not_sure: "Not sure"
+      errors:
+        one_required: "You must select an answer"
+    groups:
+      paying_bills: "Paying bills"
+      getting_food: "Getting food"
+      not_in_work: "Being unemployed or not having any work"
+      going_in_to_work: "Going in to work"
+      feeling_unsafe: "Feeling unsafe"
+      housing: "Having somewhere to live"
+      mental_health: "Mental health and wellbeing"
+    questions:
+      are_you_out_of_employment:
+        title: "Have you been made unemployed or put on temporary leave (furloughed)?"
+      are_you_non_keyworker_still_going_to_work:
+        title: "Are you still going in to work even though you’re not a key worker?"
+        special_options:
```

Comment:
As mentioned above, to keep it easier to read and update, I think each question should define all of it's options.

Reply:
main discussion on [this thread](https://github.com/alphagov/govuk-coronavirus-find-support/pull/26#discussion_r405456524)